sympy/polys/rings.py

```diff
@@ -1953,20 +1954,44 @@ def imul_num(p, c):
             p[exp] *= c
         return p
 
-    def content(f):
-        """Returns GCD of polynomial's coefficients. """
-        domain = f.ring.domain
-        cont = domain.zero
+    def content(self, domain):
```

Comment:
There is an error that the` list has no attribute content`
Where should I add this method to the list type?